src/hooks/useUpdateRecentSeatedUsers.ts

```diff
@@ -18,34 +19,63 @@ import { getCurrentTimeInMilliseconds } from "utils/time";
 
 import { useUser } from "hooks/useUser";
 
-export const useUpdateRecentSeatedUsers = <T extends VenueTemplate>(
+const updateSeatedData = async <T extends VenueTemplate>(
+  template: T,
+  venueId: string | undefined,
+  userId: string | undefined,
+  venueSpecificData: RecentSeatedUserData<T>["venueSpecificData"] | FalseyValue
+) => {
+  if (!venueSpecificData || !venueId || !userId) return;
+  console.log("updating");
```

Comment:
REmove this too?

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2361/commits/81f321955aeab6f1da1596d30eb3fd572a53fb14